### ISSUE TITLE
Kitty exits when dragging Firefox tab over it

### ISSUE THREAD

**Issue by florianb** [kovidgoyal/kitty#2668]
When running Kitty (no matter which config) and drag a Firefox tab over it, Kitty immediately exits.

### Steps to reproduce the behavior

1. Open a instance of Kitty & Firefox
2. Drag a tab out of Firefox
3. Drag the tab over the Kitty window
4. Kitty exits

### Expected behaviour

Kitty should not exit.

### Enviroment details
OS: Ubuntu 20.04 LTS

```
kitty 0.17.2 (b65119b8a9) created by Kovid Goyal
Linux onyx 5.4.0-29-generic #33-Ubuntu SMP Wed Apr 29 14:32:27 UTC 2020 x86_64
Ubuntu 20.04 LTS \n \l
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04 LTS"
Loaded config files: /etc/xdg/kitty/kitty.conf, <<snip>>/.config/kitty/kitty.conf
[140 20:36:19.477652] Ignoring unknown config key: Tab
[140 20:36:19.477765] Ignoring invalid config line: }}}
Running under: X11

Config options different from defaults:
allow_remote_control  y
cursor_shape          2
font_family           Fira Code
symbol_map            {(10140, 10140): 'Symbola'}
update_check_interval 0.0
Added shortcuts:
	control+alt+enter KeyAction(func='new_window_with_cwd', args=())
	shift+control+alt+page_up KeyAction(func='move_window_forward', args=())
	shift+control+alt+page_down KeyAction(func='move_window_backward', args=())
Changed shortcuts:
	shift+control+t KeyAction(func='new_tab_with_cwd', args=['!neighbor'])
	shift+control+page_up KeyAction(func='next_window', args=())
	shift+control+page_down KeyAction(func='previous_window', args=())
```

### Additional context

- The problem also occurs with `--config NONE`
- The problem occurs with Firefox Web Developer Edition 77.0b7 (64-bit)
- The problem occurs with Firefox Standard Edition 76.0.1 (64-Bit)
- The problem does not occur with Chrome

---

If there's anything i can do, please let me know.

**Comment by Luflosi** [comment]
This is fixed in 0.17.3, see #2505.

**Comment by florianb** [comment]
Nice thanks - i'll update ASAP... :smile: 
Sorry for the noise.
